src/pages/SalesLeadDeliveryPage/SalesLeadDeliveryPage.tsx

```diff
@@ -613,35 +691,35 @@ const ConfirmSection: React.FC<{
           },
   }
 
-  const { data: leadCandidatesData, loading: isLeadCandidatesLoading } = useQuery<
-    hasura.GET_LEAD_CANDIDATES_AGGREGATE,
-    hasura.GET_LEAD_CANDIDATES_AGGREGATEVariables
-  >(
-    gql`
-      query GET_LEAD_CANDIDATES_AGGREGATE($condition: member_bool_exp) {
-        member_aggregate(where: $condition) {
-          aggregate {
-            count
-          }
-        }
-      }
-    `,
-    {
-      fetchPolicy: 'no-cache',
-      variables: {
-        condition: leadCandidatesCondition,
-      },
+  const { data: leadCandidatesData, loading: loadingLeadCandidates } = useQuery<
+    hasura.GetLeadCandidates,
+    hasura.GetLeadCandidatesVariables
+  >(GetLeadCandidates, {
+    fetchPolicy: 'no-cache',
+    variables: {
+      condition: leadCandidatesCondition,
     },
-  )
-
-  const filteredMemberIdCount = useMemo(() => {
-    const count = leadCandidatesData?.member_aggregate.aggregate?.count || 0
-    return count
-  }, [leadCandidatesData?.member_aggregate])
-
-  const isLoading = isLeadCandidatesLoading
+  })
 
-  const { id: appId, settings } = useApp()
+  const leadCandidatesCounts = leadCandidatesData ? leadCandidatesData.member.length : 0
+
+  const members = leadCandidatesData?.member.slice(0, limit).map(m => ({
+    id: m.id,
+    pictureUrl: m.picture_url || '',
+    name: m.name,
+    email: m.email,
+    role: m.role as 'general-member' | 'content-creator' | 'app-owner',
+    createdAt: new Date(m.created_at),
+    username: m.username,
+    loginedAt: m.logined_at,
+    managerId: m.manager_id || '',
+    star: m.star,
+    lastMemberNoteCalled: m.last_member_note_called ? moment(m.last_member_note_called).format('YYYY-MM-DD') : '',
+    lastMemberNoteAnswered: m.last_member_note_answered ? moment(m.last_member_note_answered).format('YYYY-MM-DD') : '',
+    closedAt: m.closed_at ? moment(m.closed_at).format('YYYY-MM-DD') : '',
+    completedAt: m.completed_at ? moment(m.completed_at).format('YYYY-MM-DD') : '',
+    recycledAt: m.recycled_at ? moment(m.recycled_at).format('YYYY-MM-DD') : '',
+  }))
```

Comment:
without filter condition？

Reply:
The purpose here is to get 50 pieces of member data from leadCandidatesData. It has nothing to do with the filter condition.

Reply:
oh, I didn't realize it only needs to fetch 50 records. 
So, does this mean that the feature does not need to use the filter functionality for these 50 records?

Reply:
I originally deconstructed data because typescript prompts about data missing types.
I changed the data of currentMembers props to define the type and simplified the writing here.